app/repositories/maps.py

```diff
@@ -122,7 +122,7 @@ async def create(
 
 
 async def fetch_one(
```

Comment:
not sure about this for fetch_one, with the composite key, you could have overlapping beatmap ids between two servers

Reply:
interesting take actually

Reply:
you may want to check our current usage of `server="osu!"` throughout the codebase; i made them without this consideration

Reply:
@tsunyoku curious of your thoughts here (if any)

Reply:
> you may want to check our current usage of `server="osu!"` throughout the codebase; i made them without this consideration

Yeah I actually checked that, especially because it got quite irritating in our implementation of above and I needed to change it everywhere :kekw:.

I can include pr with changes of that  in beatmaps.py and other places if needed.

Reply:
> you may want to check our current usage of `server="osu!"` throughout the codebase; i made them without this consideration

wasn't that for when eventually you would implement bss? atleast that's what I use it for

Reply:
@def750 @cmyui little bit confused about your guys' discussion.. we cannot have overlapping beatmap ids across servers because you can't possibly handle 2 maps with the same id in the osu! client.

i don't think this PR is an issue, but i don't really understand the comments about overlapping ids either. all this pr does, in my eyes, is allow you to fetch from both servers simultaneously (may be useful for direct if we ever have bss?)

Reply:
okay yeah @tsunyoku i'm moving in that direction now #358